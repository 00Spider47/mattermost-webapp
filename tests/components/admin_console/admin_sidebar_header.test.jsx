// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
// See License.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import AdminSidebarHeader from 'components/admin_console/admin_sidebar_header/admin_sidebar_header.jsx';

describe('components/AdminSidebarHeader', () => {
    const baseProps = {
        user: {id: 'someid', last_picture_update_at: 1234},
    };

    test('should match snapshot', () => {
        const wrapper = shallow(
            <AdminSidebarHeader {...baseProps}/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
