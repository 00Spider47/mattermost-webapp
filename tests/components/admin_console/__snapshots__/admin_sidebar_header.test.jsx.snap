// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AdminSidebarHeader should match snapshot 1`] = `
<div
  className="team__header theme"
>
  <a
    href="#"
    onClick={[Function]}
  >
    <div
      className="header__info"
    >
      <div
        className="team__name"
      >
        <FormattedMessage
          defaultMessage="System Console"
          id="admin.sidebarHeader.systemConsole"
          values={Object {}}
        />
      </div>
      <div
        className="user__name"
      >
        @undefined
      </div>
    </div>
  </a>
  <Connect(AdminNavbarDropdown) />
</div>
`;
